cypress/e2e/cross-chain/cross_chain_c_to_x_mock.cy.ts

```diff
@@ -0,0 +1,247 @@
+import {
+    BN,
+    bnToAvaxX,
+    bnToBigAvaxC,
+    bnToBigAvaxX,
+    GasHelper,
+    Big,
+    numberToBNAvaxC,
+} from '@c4tplatform/camino-wallet-sdk'
```

Comment:
As mentioned in my previous comments, this should have `from '@c4tplatform/camino-wallet-sdk/dist'`

Reply:
thx, I've solved it.